### DISCUSSION TITLE
Being able to limit a BelongsToManyMultiSelect

### DISCUSSION THREAD

**Discussion by stefanfortuin** [filamentphp/filament#2294] [Ideas]
In my project i have it setup in the database that there is a pivot table relationship.
But for now only one option should be selected, not multiple.

It would be handy to be able to use a BelongsToManyMultiSelect in such a way to limit it to a certain amount of selections.
Maybe if there is a limit of 1, make it look like a normal select component.

**Comment by danharrin** [comment]
We have `maxItems(1)` for that, right? When #2057 is merged, the max items will be enforced on the frontend as well as through backend validation.

**Comment by stefanfortuin** [comment]
Ah okay.

I was confused because maxItems() and minItems() in the docs are only mention for Repeaters and Builders.
So I didn't know that it was usable for Select inputs too.

Maybe a thing to mention in the docs.
